module/VuFind/src/VuFind/View/Helper/AbstractSearch.php

```diff
@@ -104,6 +104,7 @@ public function renderSpellingSuggestions($msg, $results, $view)
                     $html .= $this->renderExpandLink($url, $view);
                 }
             }
+            $html .= '<br>';
```

Comment:
I don't really like the `<br>`s, either in terms of layout (the line could be wrapped in a div or otherwise display:block) or semantically (should be `<li>`s, even if visually list-style: none.  But none of that is really relevant to this PR fixing the headings.

Reply:
Agreed, I thought about changing this but then decided to avoid scope-creep. :-) Please feel free to open a follow-up PR if you'd like to clean this up. It's worth doing but I am unlikely to get there any time soon.